assets/dev/js/editor/container/container.js

```diff
@@ -238,28 +238,43 @@ export default class Container extends ArgsObject {
 		}
 	}
 
-	addRepeaterItem( repeaterName, rowSettingsModel, index ) {
-		let rowId = rowSettingsModel.get( '_id' );
+	/**
```

Comment:
It looks like this file doesn't require any change to fix the issue,
Please send unrelated code via a separated PR.

Reply:
The only 'logic' that changed in code is that it now does return the new container, are you sure we need separate PR for that?

Reply:
Yes, a PR should contain only the changes that needed, in order to reflect exactly how the bug was solved.

Reply:
Handled.